tests/jeos/firstrun.pm

```diff
@@ -70,8 +70,16 @@ sub run {
 
     # JeOS on generalhw
     mouse_hide;
+    # expected in near future if (is_tumbleweed || is_leap('15.4+') || is_sle('15-sp4+') {
+    unless (is_leap('15.4+') || is_sle('15-sp4+')) {
```

Comment:
I think this is wrong

Reply:
Yup, copy&paste. Should be fixed and comment added. Thanks